app/src/main/java/com/carminacotiga/newsreader/ArticleDetails.java

```diff
@@ -0,0 +1,30 @@
+package com.carminacotiga.newsreader;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.carminacotiga.newsreader.ui.feature.adapter.ArticleAdapter;
+
+public class ArticleDetails extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_article_details2);
+
+        TextView titleTextView = findViewById(R.id.detailsTitle);
+        TextView contentTextView = findViewById(R.id.detailsContent);
```

Comment:
we don't do findViewById anymore, either ViewBinding or DataBinding. In our case let's use DataBinding with layout and specific view model, examples here: https://developer.android.com/topic/libraries/data-binding/expressions#binding_data